src/VisualStudio/IntegrationTest/IntegrationTests/Extensions/DebuggerExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Roslyn.VisualStudio.IntegrationTests.Extensions.Editor;
+
+namespace Roslyn.VisualStudio.IntegrationTests.Extensions.Debugger
+{
+    public static partial class DebuggerExtensions
```

Comment:
Ugh, really not a fan of the extension pattern here. I would have expected the pattern that you could do something like VisualStudio.Debugger.SetBreakpoint(), etc. Just have separate classes to divide up the functionality.